pkgs/applications/emulators/mame/default.nix

```diff
@@ -138,21 +140,32 @@ stdenv.mkDerivation rec {
     runHook preInstall
 
     make -f dist.mak PTR64=${lib.optionalString stdenv.is64bit "1"}
+
+    # mame
     mkdir -p ${dest}
     mv build/release/*/Release/mame/* ${dest}
 
-    mkdir -p $out/bin
-    find ${dest} -maxdepth 1 -executable -type f -exec mv -t $out/bin {} \;
-    install -Dm755 src/osd/sdl/taputil.sh $out/bin/taputil.sh
-
-    installManPage ${dest}/docs/man/*.1 ${dest}/docs/man/*.6
+    install -Dm755 mame -t $out/bin
     install -Dm644 ${icon} $out/share/icons/hicolor/scalable/apps/mame.svg
+    installManPage ${dest}/docs/man/*.1 ${dest}/docs/man/*.6
 
     mv artwork plugins samples ${dest}
 
+    # mame-tools
+    for _i in castool chdman floptool imgtool jedutil ldresample ldverify nltool nlwav pngcmp regrep romcmp \
+              split srcclean testkeys unidasm; do
+      install -Dm755 $_i -t $tools/bin
+    done
+    mv $tools/bin/{,mame-}split
+
     runHook postInstall
   '';
 
+  postFixup = ''
+    mkdir -p $tools/share/man
+    mv {$out,$tools}/share/man/man1
```

Comment:
Absolutely all manuals are from `$tools`? 
(Or it was solved before this line?)

Reply:
All `man1` manuals are from tools. The manual from `mame` itself is in `man6` and it is still there.